source/languages/en/riak/dev/using/data-types.md

```diff
@@ -1636,7 +2168,7 @@ We can remove that interest in just the way that we would expect:
 
 SetUpdate su = new SetUpdate().remove("tango dancing");
 MapUpdate annikaUpdate = new MapUpdate()
-        .update("widget_purchases", su);
+        .update("interests_set", su);
```

Comment:
At :2085 `widget_purchases` was changed to `interests` rather than `interests_set`. Is this intentional?